content/en/tracing/setup/nodejs.md

```diff
@@ -52,7 +52,9 @@ Finally, import and initialize the tracer:
 
 ```js
 // This line must come before importing any instrumented module.
-const tracer = require('dd-trace').init()
+const tracer = require('dd-trace').init({
+  service: '<SERVICE_NAME>',
```

Comment:
This is not a required parameter.

Reply:
From reading the original issue, it looks like there is a bug since the service name should automatically added. I would recommend opening an issue in https://github.com/DataDog/dd-trace-js